app/support/auto_expire_reservation.rb

```diff
@@ -15,7 +15,8 @@ def order_details
 
   def purchased_active_order_details
     OrderDetail.purchased_active_reservations
-      .merge(Reservation.relay_in_progress)
+      .joins("INNER JOIN products ON order_details.product_id = products.id")
+      .joins("INNER JOIN relays ON relays.instrument_id = products.id")
```

Comment:
Would `.joins(:products).joins(:relays)` work? Is there a better way we can filter out res-only instruments? This just seems a little too SQL-y for here.